doc/data-retrieval.rst

```diff
@@ -237,6 +238,20 @@ Data in this section is retrieved and extracted in rules specified in ``rules/re
 - **License:** GPL-3.0
 - **Description:** Contains matched dataset of powerplants in Europe.
 
+``data/busshapes/bidding_zones_electricitymaps.geojson``
+
+- **Source:** Electricity Maps, https://github.com/electricitymaps/electricitymaps-contrib
+- **Link:** https://raw.githubusercontent.com/electricitymaps/electricitymaps-contrib/v1.238.0/web/geo/world.geojson
+- **License:** GNU-AGPLv3
```

Comment:
```suggestion
- **License:** AGPL-3.0
```

nit: that acronym is more common and also what GitHub uses when it shows licenses in the repos